jeffrey/make_custom_table.sql

```diff
@@ -0,0 +1,16 @@
+create table jeffreys_table (
+    name varchar(500),
+    favorite_food varchar(30) CHECK (
+        favorite_food in (
+            'pizza',
+            'ramen',
+            'baguette',
+            'beans and toast'
```

Comment:
These are the allowable values in the field, as a result of the `CHECK` constraint. That's right, these are the only favorite foods lol